agent_verification/src/connection/command.rs

```diff
@@ -0,0 +1,15 @@
+use oid4vc_core::authorization_response::AuthorizationResponse;
+use serde::Deserialize;
+use siopv2::siopv2::SIOPv2;
+
+use super::aggregate::SIOPv2AuthorizationRequest;
+
+#[derive(Debug, Deserialize)]
+#[serde(untagged)]
+pub enum ConnectionCommand {
+    VerifySIOPv2AuthorizationResponse {
+        siopv2_authorization_request: SIOPv2AuthorizationRequest,
+        siopv2_authorization_response: AuthorizationResponse<SIOPv2>,
+        connection_notification_uri: Option<url::Url>,
```

Comment:
What is the use case for a `connection_notification_uri`?

Reply:
As discussed internally, I've now removed this functionality as we will need to implement some sort of outbound adapter that listens for a `SIOPv2AuthorizationResponseVerified` event. This will be added in a later PR.